bundle/compliance/cis_gcp/rules/cis_3_8/rule.rego

```diff
@@ -0,0 +1,21 @@
+package compliance.cis_gcp.rules.cis_3_8
+
+import data.compliance.lib.common
+import data.compliance.policy.gcp.data_adapter
+import future.keywords.if
+
+finding = result if {
+	data_adapter.is_subnetwork
+
```

Comment:
I think we should add an additional filter.
> Note: Subnets reserved for use by internal HTTP(S) load balancers do not support VPC flow logs.

`not data_adapter.resource.data.purpose == "INTERNAL_HTTPS_LOAD_BALANCER"`